nominees/public-lab.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Public Lab",
+  "aliases": [
+    "The Public Laboratory for Open Technology and Science",
+    "PLOTS"
+  ],
+  "description": "Public Lab is a worldwide community that develops and applies open-source tools for environmental exploration and investigation. Public Lab works to build a healthier and more equitable world through community science, open technology, and environmental justice. Public Lab supports local environmental problem owners to draw upon expertise from a distributed global community when crafting research questions, through designing and implementing accessible monitoring techniques, to developing advocacy pathways.",
```

Comment:
The language here should be for PLOTS, not `PublicLab.org`. We can ask for their approval on this text from the GitHub README (or maybe @ebarry can spot a quick check here in the comments):

```suggestion
  "description": "The content management system for the Public Lab research community, the plots2 web application is a combination of a group research blog -what we call \"research notes\"-and a wiki. It showcases a variety of features that help the Public Lab community collaborate on environmental technology, design, documentation, and community organizing. Originally a Drupal site, it was rewritten in 2012 with Ruby on Rails.",
```